force-app/main/default/classes/ElevateAuthorizedGift.cls

```diff
@@ -29,12 +29,12 @@
  */
 public with sharing class ElevateAuthorizedGift {
     @AuraEnabled public ElevateTokenizedGift tokenizedGift;
-    @AuraEnabled public String paymentId;
+    @AuraEnabled public String paymentId {get; set;}
     @AuraEnabled public DateTime authExpiration;
     @AuraEnabled public String status;
     @AuraEnabled public String paymentMethod;
     @AuraEnabled public String originalTransactionId;
-    @AuraEnabled public String elevateBatchId;
+    @AuraEnabled public String elevateBatchId {get; set;}
```

Comment:
What is the reason for adding the getter/setters to these two fields?

Reply:
Same with above, this was related to the deserialization of an ElevateAuthorizedGift. Without the getter/setters, elevateBatchId and elevatePaymentId would not be set when passing the object from LWC to the server.